packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -127,5 +133,20 @@
       <artifactId>parquet-avro</artifactId>
       <scope>compile</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <scope>${avro.scope}</scope>
+    </dependency>
   </dependencies>
+
+  <profiles>
+    <profile>
+      <id>shade-avro</id>
```

Comment:
same here.. have the bundle name in the profile id? and probably move this to root poom as may be `<id>aws-emr-profile</id>` , that way you can control other overrides as well